app/assets/stylesheets/common.css.scss

```diff
@@ -73,3 +73,7 @@ body {
 .invoice-title {
   color: $title-invoice-color
 }
+
+#new_invitation {
```

Comment:
@yunixon it's a usual ID for form. Please use something more specific for the link, like `#invitation_link` or `#invitation_link_area`.